### ISSUE TITLE
False positive `Layout/EmptyLinesAroundArguments` with multiline heredoc args

### ISSUE THREAD

**Issue by jacobpgn** [bbatsov/rubocop#5339]
`Layout/EmptyLinesAroundArguments` appears to fail incorrectly when a method is called with both:
1. a multiline heredoc as an argument (or part of an argument, e.g. as a hash value)
2. a chained method which also receives arguments

## Expected behavior

I would expect the following to not fail for this cop:

```ruby
def chocolate
  bar(<<~DOCS)
    Sometimes we foo the bars

    Other times the bars foo us
  DOCS
    .call!(true)
end
```

## Actual behavior

`C: Layout/EmptyLinesAroundArguments: Empty line detected around arguments` 💥 

## Steps to reproduce the problem

See example in _Expected behaviour._

## RuboCop version

```
0.52.1 (using Parser 2.4.0.2, running on ruby 2.5.0 x86_64-darwin16)
```
